epub33/rs/index.html

```diff
@@ -2068,7 +2068,7 @@ <h3>Description</h3>
 					<pre>alert("Reading System name: " + navigator.epubReadingSystem.name);</pre>
 				</aside>
 
-				<p>Reading Systems MUST expose the <code>epubReadingSystem</code> object on the <code>navigator</code>
+				<p id="scripting-req-epubReadingSystem" data-tests="#scripting-epubReadingSystem-support,#scripting-epubReadingSystem-support_svg,#scripting-epubReadingSystem-support_iframe,#scripting-epubReadingSystem-support_iframe_svg">Reading Systems MUST expose the <code>epubReadingSystem</code> object on the <code>navigator</code>
```

Comment:
I haven't seen any other IDs that use mixed case, so I don't think we should start here. Note that line 865 is confreq-rs-scripted-readingsystem-object.